subprojects/packagefiles/orcus/src/liborcus/meson.build

```diff
@@ -0,0 +1,107 @@
+cdata = configuration_data()
+cdata.set('ORCUS_MAJOR_VERSION', version_major)
+cdata.set('ORCUS_MINOR_VERSION', version_minor)
+cdata.set('ORCUS_MICRO_VERSION', version_micro)
+
+
+configure_file(input: 'constants.inl.in',
+  output: 'constants.inl',
+  configuration: cdata)
+
+orcus_lib = library('orcus-0.17',
+  # core
```

Comment:
These should probably all have `version: '0.0.0'` corresponding to upstream. Silly that they have a useless soname on top of having a version in the library name, but what can you do...

Reply:
FWICT this library has exaclty one user, and that is Libreoffice. They use system libs on Linux so this would only really be used on Windows and Mac so this does not really matter.